notebooks/5. Compute sidewalk width.ipynb

```diff
@@ -131,7 +132,25 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fb577bc3",
+   "id": "b3d9e7fe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_points_on_line(line, max_seg_length): \n",
+    "    distance_delta = max_seg_length   \n",
+    "    # generate the equidistant points\n",
+    "    distances = np.arange(0, line.length, distance_delta)\n",
+    "    points = sg.MultiPoint([line.interpolate(distance) for distance in distances] + [(line.coords[0], line.coords[-2])])\n",
```

Comment:
Is it me or (line.coords[0], line.coords[-2]) is just the beginning of the line? line coords is just 2 points, so line.coords[0] = line.coords[-2] :/ 
In any case, does it make more sense to put that part on a separate line and give it a meaningful name so it's clear what that is?

Reply:
You are right, it is a duplicate and I don't know why it was there. Removed it, tested it and pushed.